### PR TITLE
feat(epwmap): refactoring and CD

### PR DESCRIPTION
**Pull request by AntonelloDN**
Hi @mostaphaRoudsari, This PR should automatise part of the process of the generation of the EPW map. You should just set the tokens and their permissions which are required for actions.
Once done you should get something like [this](https://antonellodn.github.io/epwmap/)

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+on:
+  push:
+    branches: [ master ]
+    paths-ignore:
+      - '**/README.md'
+  pull_request:
+    branches: [ master ]
+    paths-ignore:
+      - '**/README.md'
```
Comment (mostaphaRoudsari) [inline .github/workflows/release.yaml:11]: We probably don't want to release the map on the pull request. We should only limit this to push to the master branch.

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        run: npm i
+
+      - name: Build components
+        run: npm run build
+
+      - name: Deploy
+        uses: peaceiris/actions-gh-pages@v3
+        if: ${{ github.ref == 'refs/heads/master' }}
+        with:
+          personal_token: ${{ secrets.PER_TOKEN }}
```
Comment (mostaphaRoudsari) [inline .github/workflows/release.yaml:41]:
```suggestion
          personal_token: ${{ secrets.DEPS_UPDATING}}
```

File: README.md [hunk 3]
```diff
@@ -1,13 +1,9 @@
...
 
-Credits
-=======
-Weather data are hosted by Department of Energy > http://apps1.eere.energy.gov/buildings/energyplus/weatherdata_about.cfm
-
-Weather data information is parsed by Ladybug https://github.com/mostaphaRoudsari/ladybug
-
-This page in using D3 library href="https://github.com/mbostock/D3 Check this link for mroe examples: http://d3js.org/
+## Credits
+* Weather data are hosted by Department of Energy(https://energyplus.net/weather)
```
Comment (mostaphaRoudsari) [inline README.md:7]:
```suggestion
* Weather data are hosted by the [Department of Energy](https://energyplus.net/weather) and [OneBuilding](Climate.OneBuilding.Org)
```

**Review by mostaphaRoudsari — COMMENTED** [review]
Hi @AntonelloDN! Thank you for the PR. See my comments below.

File: .github/workflows/release.yaml (hunk 1 above)
Comment (AntonelloDN) [reply .github/workflows/release.yaml]: Done!

**Pull request merged**